server/opts_test.go

```diff
@@ -1155,6 +1155,39 @@ func TestPanic(t *testing.T) {
 	}
 }
 
+func TestPingIntervalOld(t *testing.T) {
+	conf := createConfFile(t, []byte(`ping_interval: 5`))
+	defer os.Remove(conf)
+	opts := &Options{}
+	err := opts.ProcessConfigFile(conf)
+	if err == nil {
+		t.Fatalf("expected an error")
+	}
+	if err, ok := err.(*processConfigErr); !ok {
+		t.Fatalf("expected an error of type processConfigErr")
+	} else if len(err.warnings) != 1 {
+		t.Fatalf("expected processConfigErr to have one warning")
+	} else if len(err.errors) != 0 {
+		t.Fatalf("expected processConfigErr to have no error")
+	}
+	if opts.PingInterval != 5*time.Second {
+		t.Fatalf("expected ping interval to be 5 seconds")
+	}
+}
+
+func TestPingIntervalNew(t *testing.T) {
+	conf := createConfFile(t, []byte(`ping_interval: "5m"`))
+	defer os.Remove(conf)
+	opts := &Options{}
+	err := opts.ProcessConfigFile(conf)
```

Comment:
Again, styling and no big deal, but here you could have made it a "one liner". Also, worth reporting the error if one is raised:
```
if err := opts.ProcessConfigFile(conf); err != nil {
    t.Fatalf("unexpected error: %v", err)
}
if opts.PingInterval ...
```

Reply:
true that, will change. Is there generally a preference one way or the other?

Reply:
Yes, the preference is that if you don't need the variable outside of the "if", make it a "one liner". If you do, we usually declare outside of the "if" to avoid those "else" statements because of the scope. So this:
```
v, err := someFunc()
if err != nil {
  return err
}
// Rest of the code use "v" in some places.
```

Reply:
changed all but the one liner cases